.github/workflows/main-docker-build.yml

```
@@ -0,0 +1,50 @@
+name: "main-docker-build"
+
+on:
+  push:
```

Let's add Pull requests so we can combine the workflows
```suggestion
  pull_request:
  push:
```